docs/rules/no-negated-condition.md

```diff
@@ -1,34 +1,12 @@
-# Disallow use of negated expressions in conditions (no-negated-condition)
+# disallow negated conditions (no-negated-condition)
 
-Checks against the use of a negated expression in an if condition when the else branch is not empty or in a ternary operator. Negated conditions are more difficult to understand. Code can be made more readable by inverting the condition instead.
-
-For example:
-
-```js
-if (!a) {
-    doSomething();
-}
-else {
-    doSomethingElse();
-}
-```
-
-should instead be written as:
-
-```js
-if (a) {
-    doSomethingElse();
-}
-else {
-    doSomething();
-}
-```
+Checks against the use of a negated expression in an `if` condition when the `else` branch is not empty or in a ternary operator. Negated conditions are more difficult to understand. Code can be made more readable by inverting the condition instead.
 
 ## Rule Details
 
-The rule is aimed at preventing the use of a negated expression in a condition.
+The rule disallows negated conditions.
 
-The following patterns are considered warnings:
+Examples of **incorrect** code for this rule:
```

Comment:
Minor nit: it looks like there is was an extra empty line preceding the ternary expression that we could delete.